util/metrics/import_metrics_files.rb

```diff
@@ -63,6 +65,19 @@ def import_metrics_files
       end
     end
 
+    # Returns true if the specified dir is a GPLT results dir, otherwise false
+    #
+    # @author Bill Claytor
+    #
+    # @return [Boolean] true if the specified dir is a GPLT results dir, otherwise false
+    #
+    # @example
+    #   result = valid_results_dir?(results_dir)
+    #
+    def valid_results_dir?(results_dir)
+      results_dir.include?("PERF_")
```

Comment:
Updated to use the following since `results_dir` is the full path and it also needs to work for scale results:
```
File.basename(results_dir).match?(/^PERF_(SCALE_)?\d+$/)
```